src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ExpanderTests/Expander_Bindings_TemplatedControl.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<UserControl
+    x:Class="UITests.Shared.Microsoft_UI_Xaml_Controls.ExpanderTests.Expander_Bindings_TemplatedControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Shared.Microsoft_UI_Xaml_Controls.ExpanderTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+	<UserControl.Resources>
+		<ResourceDictionary>
+			<Style BasedOn="{StaticResource DefaultSettingsExpanderStyle}"
```

Comment:
could just use the style explicitly down below

Reply:
Removed style resources that were not defined